src/main/java/com/schematica/client/util/RotationHelper.java

```diff
@@ -27,6 +27,10 @@
 
 public final class RotationHelper
 {
+    /**
+     * This is needed for Schematica to work.
+     */
+    @SuppressWarnings("deprecation")
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Suppressing warnings is not allowed [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1309)